features/all-positions/PositionActionsDialog.tsx

```diff
@@ -230,7 +234,7 @@ const PositionActionsDialog = (props: DialogProps) => {
         setHash(null);
         setSuccess(null);
         setError(null);
-        const collateralToDepositWei = toWei(collateralToDeposit);
+        const collateralToDepositWei = toWei(collateralToDeposit, collDecs);
```

Comment:
did not know about this function, that seems useful. so what is this doing exactly? the collateraltodeposit is a number entered by the user as float, and this converts it to wei in its native decimals?

Reply:
Yep exactly, where `collDecs = 8` if the collateral currency uses 8 decimal precision